vassal-app/src/main/java/VASSAL/tools/image/ImageUtils.java

```diff
@@ -463,4 +467,60 @@ public static String stripImageSuffix(String name) {
     }
     return name;
   }
+
+  /**
+   * Create a viewable representation of a null or empty image to
+   * use as a place holder in Configurers.
+   *
+   * The image will contain the translated text for the key
+   * Editor.ImageUtils.no_image
+   *
+   * @return Viewable null image
+   */
+  public static Image createViewableNullImage() {
+    return createViewableNullImage(64, 64);
+  }
+
+  /**
+   * Create a viewable representation of a null or empty image to
+   * use as a place holder in Configurers.
+   * *
+   * The image will contain the translated text for the key
+   * Editor.ImageUtils.no_image
+   *
+   * @param minWidth Minimum width for generated the image
+   * @param height   Height of the generated image
+   * @return Viewable null image
+   */
+  public static Image createViewableNullImage(int minWidth, int height) {
+
+    final int FONT_SIZE = 12;
+    final Font FONT = new Font(Font.DIALOG, Font.ITALIC, FONT_SIZE);
+
+    // Determine the size of the translated string
+    final String s = Resources.getString("Editor.ImageUtils.no_image");
+    Image image = new BufferedImage(minWidth, height, BufferedImage.TYPE_4BYTE_ABGR);
```

Comment:
We've had graphics glitches when we've tried to paint image types other than the "compatible" one. It would be better to create this image using `createCompatibleTranslucentImage(minWidth, height)` to avoid that.

Reply:
Also: As you're using this image just to get a Graphics2D from it, it's size doesn't matter; you can use the pre-existing `compatTransImage` static class member to get your Graphics2D for text sizing and avoid creating this BufferedImage.

Reply:
Fixed.